src/filesystem-cache.ts

```diff
@@ -49,7 +49,7 @@ export class FilesystemCache {
     this.cacheConfig = this.config.cacheConfig;
 
     if (!fs.existsSync(this.getDir(''))) {
-      fs.mkdirSync(this.getDir(''));
+      fs.mkdirSync(this.getDir(''), { recursive: true });
```

Comment:
`src/filesystem-cache.ts` is failing. Because `fs.mkdirSync` does not create a directory recursively by default.